shared/teams/new-team/wizard/create-channels.tsx

```diff
@@ -134,19 +135,17 @@ const styles = Styles.styleSheetCreate(() => ({
   addButton: Styles.platformStyles({
     isElectron: {width: 42},
     isMobile: {width: 47},
+    isTablet: {alignSelf: 'flex-start'},
   }),
   banner: Styles.platformStyles({
     common: {backgroundColor: Styles.globalColors.blue, height: 96},
     isElectron: {overflowX: 'hidden'},
   }),
-  body: Styles.platformStyles({
-    common: {
-      ...Styles.padding(Styles.globalMargins.small),
-      backgroundColor: Styles.globalColors.blueGrey,
-    },
-    isElectron: {minHeight: 326},
-    isMobile: {...Styles.globalStyles.flexOne},
-  }),
+  body: {
+    ...Styles.padding(Styles.globalMargins.small),
+    backgroundColor: Styles.globalColors.blueGrey,
```

Comment:
:sheep: putting this on `Modal`'s `backgroundStyle` prop makes it so scroll bouncing doesn't have a background color edge.